src/fusor/translator.py

```diff
@@ -955,12 +955,59 @@ async def translate(
 class CIVICTranslator(Translator):
     """Initialize CIVICTranslator"""
 
-    async def translate(self, civic: CIVIC) -> CategoricalFusion:
+    def _get_breakpoint(
+        self, coordinate_data: ExonCoordinate, is_5prime: bool = True
+    ) -> int:
+        """Extract correct breakpoint for downstream processing
+
+        :param coordinate_data: An ExonCoordinate object
+        :param is_5prime: A boolean indicating if 5'partner is being processed
+        :return: The modified genomic breakpoint, taking strand, offset, and
+            offset direction into account
+        """
+        if is_5prime:
+            coord_to_use = (
+                coordinate_data.stop
+                if coordinate_data.strand == "POSITIVE"
```

Comment:
Could we use or create an enum instead of harcoding?

Reply:
I could create an enum here, but I think it would have to be a separate enum from the Cool-Seq-Tool strand enum?